docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing BKPR
```

Comment:
I would title this `Install` instead of `Installing`, using a direct "action verb" is more impactful.